editor/editor_inspector.cpp

```diff
@@ -2469,7 +2469,7 @@ void EditorInspector::_notification(int p_what) {
 		}
 		if (refresh_countdown > 0) {
 			refresh_countdown -= get_process_delta_time();
-			if (refresh_countdown <= 0) {
+			if (refresh_countdown <= 0 && !update_tree_pending) {
```

Comment:
I think this may lead to skipping a full refresh for some properties since the ones refreshed later in the `else` branch of `if (update_tree_pending)` are only the ones in the `pending` list.

But since they'd likely get picked up at the next frame it's probably not a big deal.